CMP1903_A1_2324/Testing.cs

```diff
@@ -15,6 +16,25 @@ internal class Testing
          * Use debug.assert() to make the comparisons and tests.
          */
 
-        //Method
+        /// <summary>
+        ///This class contains the "Test" method which is responsible for testing the die values generated in "Die" using the "Roll" method,
+        ///the class tests the Sum of all the die and also tests the three individual dies generated to ensure they are within the expected range.
+        /// </summary>
+
+        public void Test() //'Test' Method used for testing dies 
```

Comment:
Functional XML comments would be a great addition here too to describe the running of the function and any expected return or params.